src/webportal/src/app/job/job-view/fabric/job-detail/conn.js

```diff
@@ -222,6 +222,14 @@ export async function getContainerLog(logUrl) {
       throw new Error(`Log not available`);
     }
   } else if (config.logType === 'log-manager') {
+    if (text.length === 0) {
```

Comment:
Did you concat these 2 files? This may be better to help user, as anyway backend store 2 files.

Reply:
Front-end only return 16MB logs to end-user. And we already provide a link to log folder, user can find the full logs when he/she need more information.
Concat the two logs will cause fron-end read too much data and slow down the website.
Here we only look the rotated log when origin log  file is empty.

Reply:
If so, you should first concat (may not need real concat) then extract first latest 15M.
Otherwise, if the first file is 1B, and 2nd is 100MB, you can only show the 1B to user

Reply:
log-rotate doesn't promise not loss log. Concat the two logs may cause strange log string. We'd better give user some hint when the log is rotated.
```
copytruncate
Truncate the original log file in place after creating a copy, instead of moving the old log file and optionally creating a new one. It can be used when some program cannot be told to close its logfile and thus might continue writing (appending) to the previous log file forever. Note that there is a very small time slice between copying the file and truncating it, so some logging data might be lost. When this option is used, the create option will have no effect, as the old log file stays in place.
```

Reply:
Maybe insert one line below is better than just suddenly can only see 1B 1st file:

1st file content 
--------log is rotated, may be lost during this--------
2nd file content

Reply:
OK, will change to this

Reply:
fixed